docs/controls/AdaptiveGridView.md

```diff
@@ -10,23 +9,102 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, AdaptiveGridView,
+There are 3 ways to use this Control. 
+
+* You can set `DesiredWidth` and `ItemHeight`, which will scale the **width** of each item, and adjust the number of columns on demand, using all horizontal space.
+
+* You can set `DesiredWidth` only. This will mean that rows will take up as much space as required, using all horizontal space.
+
+* Using `OneRowModeEnabled`, you can set `DesiredWidth` and `ItemHeight`, and the **width** of each item, and the number of visible columns will be adjusted on demand, using all horizontal space.
 
 ## Syntax
 
 ```xaml
-
 <controls:AdaptiveGridView  Name="AdaptiveGridViewControl"
           ItemHeight="200"
           DesiredWidth="300"
           ItemTemplate="{StaticResource PhotosTemplate}">
 </controls:AdaptiveGridView>
-
 ```
 
 ## Example Image
 
-![AdaptiveGridView animation](../resources/images/Controls-AdaptiveGridView.gif "AdaptiveGridView")
+Using `DesiredWidth` in combination with `ItemHeight`
+
+![AdaptiveGridView DesiredHeight and ItemHeight](../resources/images/Controls/AdaptiveGridView/AdaptiveGridView-DesiredWidth&ItemHeight.gif "AdaptiveGridView DesiredHeight and ItemHeight")
+
+## Examples
+
+- Maintain aspect ratio by setting `DesiredWidth` with no `ItemHeight` set:
+
+    ![AdaptiveGridView Viewbox scaled](../resources/images/Controls/AdaptiveGridView/AdaptiveGridView-ViewboxAspectRatio.gif "AdaptiveGridView Viewbox scaled")
+
+    This still requires the `ItemTemplate` to contain some scaling logic, this can be done with Height and Width set on the content inside of a Viewbox, or using custom view logic.
+
+    - Using a `Viewbox`:
+
+        Using the `Height` and `Width` properties of content inside of a `Viewbox` means that the content inside will scale when **Width** and **Height** changes. This might not be a desired effect, and it will also likely incur a slight performance penalty.
+
+        _ItemTemplate implementation_
+
+        ```xaml
+        <DataTemplate x:Key="PhotosTemplate">
+            <Viewbox>
+                <Grid
+                    Height="300"
+                    Width="200"
+                    Background="White"
+                    BorderBrush="Black"
+                    BorderThickness="1">
+                    <Image
+                        Source="{Binding Thumbnail}"
+                        Stretch="UniformToFill"
+                        HorizontalAlignment="Center"
+                        VerticalAlignment="Center"/>
+                </Grid>
+            </Viewbox>
+        </DataTemplate>
+        ```
+
+    - To use custom view logic:
+
+        Using `MeasureOverride` on a ContentControl allows you to specify the **Width** and **Height** of the content, which might reap better performance compared to a `Viewbox`. The content inside will not scale when **Width** and **Height** changes.
```

Comment:
The phrase "The content inside will not scale" isn't exactly true. While it doesn't behave like a viewbox does, the width and height of the content does scale out making more/less content visible. If there is a RelativePanel in the template, it could add/remove content based on size. I would consider this to be scaling

Reply:
I changed it to zoom-like and no zoom ok? :)

Reply:
This is definitely a battle semantics.

Reply:
That's why I'm making comments and not requesting changes